### DISCUSSION TITLE
Apresentação

### DISCUSSION THREAD

**Discussion by DevAzarado** [NicklessOne/osib-frontend-pt#4] [Apresente-se!]
Olá a todos! Meu nome é Anderson e hoje estou aqui para aprender sobre o  desenvolvimento web e programação.

**Comment by RvRvRvV** [comment]
Opa, mn eu to fazendo curso de html5, css3 e javascript. Eu to buscando alguém que tem o mesmo conhecimento nessa área pra podermos trocar conhecimento. Você tem alguma dica para mim?
